Packs/ReversingLabs_TitaniumScale/Integrations/ReversingLabsTitaniumScale/ReversingLabsTitaniumScale.py

```diff
@@ -5,7 +5,7 @@
 import requests
 import json
 
-VERSION = "v1.0.2"
+VERSION = "v1.0.3"
```

Comment:
Does the version change have no effect on the code?

Reply:
For the TitaniumScale and TitaniumCloud apps it doesnt. In the case of these two apps, only the Docker image was changed.